data/xml/2021.acl.xml

```diff
@@ -12689,6 +12689,15 @@
       <url hash="8e86a5da">2021.acl-tutorials.1</url>
       <doi>10.18653/v1/2021.acl-tutorials.1</doi>
       <bibkey>bar-haim-etal-2021-advances</bibkey>
+      <video href="2021.acl-tutorials.1.1.mp4"/>
+      <video href="2021.acl-tutorials.1.2.mp4"/>
+      <video href="2021.acl-tutorials.1.3.mp4"/>
+      <video href="2021.acl-tutorials.1.4.mp4"/>
+      <video href="2021.acl-tutorials.1.5.mp4"/>
+      <video href="2021.acl-tutorials.1.6.mp4"/>
+      <video href="2021.acl-tutorials.1.7.mp4"/>
+      <video href="2021.acl-tutorials.1.8.mp4"/>
+      <video href="2021.acl-tutorials.1.9.mp4"/>
```

Comment:
Why are all these videos listed here under paper 1?

Reply:
Oh, I guess they are split into multiple parts?

Reply:
Yes, sometimes the tutorials are split into multiple parts.